hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -194,11 +196,27 @@ private void syncSchema(String tableName, boolean tableExists, boolean useRealTi
       String outputFormatClassName = HoodieInputFormatUtils.getOutputFormatClassName(baseFileFormat);
       String serDeFormatClassName = HoodieInputFormatUtils.getSerDeClassName(baseFileFormat);
 
+      Map<String, String> serdeProperties = ConfigUtils.toMap(cfg.serdeProperties);
+
+      // The serdeProperties is non-empty only for spark sync meta data currently.
+      if (!serdeProperties.isEmpty()) {
+        String queryTypeKey = serdeProperties.remove(ConfigUtils.SPARK_QUERY_TYPE_KEY);
+        String queryAsROKey = serdeProperties.remove(ConfigUtils.SPARK_QUERY_AS_RO_KEY);
+        String queryAsRTKey = serdeProperties.remove(ConfigUtils.SPARK_QUERY_AS_RT_KEY);
```

Comment:
This is not supported via Hoodie Delta Streamer and Hive Sync Tool right ? It also seems, that even syncing as DataSource table is not supported via Delta Streamer and Hive Sync Tool. We should plan to fix this soon before Hudi 0.9.0, to avoid inconsistent experience.

Reply:
Yes, currently the spark's properties can only synced by spark engine. If we want to add the spark properties by HiveSyncTool, we may need to push down some logical to HiveSyncTool